drizzlepac/align.py

```diff
@@ -729,18 +732,20 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, align_par
 
         # Execute checks
         nmatches_check = False
-        if num_xmatches > 4 or (num_xmatches > 2 and fit_rms_val > 0.5):
+        if num_xmatches >= align_pars['determine_fit_quality']['min_xmatches'] or \
+                (num_xmatches >= 2 and fit_rms_val > 0.5):
             nmatches_check = True
 
         radial_offset_check = False
         radial_offset = math.sqrt(
             float(item.meta['fit_info']['shift'][0])**2 +
-            float(item.meta['fit_info']['shift'][0])**2) * item.wcs.pscale  # radial offset in arssec
-        if float(num_xmatches) * 0.36 > 0.8 + (radial_offset / 10.0)**8:
+            float(item.meta['fit_info']['shift'][1])**2) * item.wcs.pscale  # radial offset in arssec
+        if float(num_xmatches + 2) * 0.36 > 0.8 + (radial_offset / 10.0)**8:
             radial_offset_check = True
```

Comment:
These look like "magic" numbers, especially the newly added "2".  If they are not going to be in a configuration file, please just add a comment as to the origin/meaning of these values.

Reply:
The change from '0' to '1' simply reflects the use of the shift in both X and Y (as needed when computing a radial value) where before the X value was simply added twice.  The '+2' was commented in ad5c3a7.